src/types/policyData.go

```diff
@@ -244,6 +244,7 @@ type KnoxMatchPaths struct {
 	ReadOnly   bool             `json:"readOnly,omitempty" yaml:"readOnly,omitempty"`
 	OwnerOnly  bool             `json:"ownerOnly,omitempty" yaml:"ownerOnly,omitempty"`
 	FromSource []KnoxFromSource `json:"fromSource,omitempty" yaml:"fromSource,omitempty"`
+	Action     string           `json:"action,omitempty"`
```

Comment:
The generated policies doesn't have the "Action" field and we were importing the type from KubeArmor. Have added it for handling action for crown jewel policies. Hope it doesn't have any bad implications.
cc: @nyrahul